src/test/suite/paths.test.ts

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
any particular reason you want strict mode with Typescript? isn't it already pretty strict?

Reply:
I found that it catches more things when compiling. Is there any harm in having it?